test/unit/StakingViewer/utils.js

```diff
@@ -0,0 +1,10 @@
+const { ethers } = require('hardhat');
+
+const ONE_DAY_SECONDS = 24 * 60 * 60;
+
+async function calculateCurrentTrancheId() {
```

Comment:
this function already exists in fork/utils.js

utils across all tests deserve a revisit / cleanup for sure, but let's avoid duplicating this for now

Reply:
I was hesitant to import something from fork dir to unit dir, hence I copied it over. but I see the point of avoiding dulpication